### PR TITLE
Document `secrets:reveal` command

### PR DESCRIPTION
**Pull request by danielburger1337**
Document the new `secrets:reveal` command.

Docs Issue: #19481 
Feature PR: https://github.com/symfony/symfony/pull/53466

### CODE REVIEW

File: configuration/secrets.rst [hunk 1]
```diff
@@ -166,6 +166,22 @@ secrets' values by passing the ``--reveal`` option:
       DATABASE_PASSWORD   "my secret"
      ------------------- ------------ -------------
 
+Reveal Existing Secrets
+-----------------------
+
+Provided that you have the **decryption key**, the ``secrets:reveal`` command allows
```
Comment (OskarStark) [inline configuration/secrets.rst]: Provided sounds wrong

Comment (danielburger1337) [reply configuration/secrets.rst]: > If you have the **decryption key**, the ``secrets:reveal`` command allows
> you to reveal a single secrets value.

Does that sound better?

Comment (OskarStark) [reply configuration/secrets.rst]: Much better, thanks

**Review by OskarStark — APPROVED** [review]
After my comment

**Comment by OskarStark** [comment]
Thank you Daniel.

**Pull request merged**